server/commands/VolumeCommand.ts

```diff
@@ -2,17 +2,40 @@ import CommandDefinition from '../domain/CommandDefinition';
 import IoConnection from '../clients/IoConnection';
 import VolumeEventMessage from "../domain/event-messages/VolumeEventMessage";
 
+const volChangeTypes: { [s: string]: string; } = {
+  '+': 'increase',
+  '-': 'decrease',
+  default: 'setter',
+};
+
+function getVolChangeType(value: string):string {
+  return volChangeTypes[value.charAt(0)] || volChangeTypes.default;
+}
+
+function getFeedbackMsg(eventMsg:VolumeEventMessage) {
```

Comment:
Prefer Message over Msg.
```suggestion
function getFeedbackMessage(eventMesssage: VolumeEventMessage) {
```